lang/python/setup_pip.py

```diff
@@ -329,31 +258,41 @@ def source_filter(sources):
         d = os.path.join(stage_dir, os.path.dirname(f))
         if not os.path.isdir(d):
             os.makedirs(d)
-        if f in movers:
-            src = movers[f]
-        else:
-            src = f
         # Symlinks are not followed in setup, we need to use real files.
-        shutil.copy2(src, os.path.join(stage_dir, f))
+        shutil.copy2(f, os.path.join(stage_dir, f))
+    # Copy files in lang/python/wiredtiger to the root folder.
+    pywt_path = os.path.join(stage_dir, "lang", "python", "wiredtiger")
+    pywt_files = os.listdir(pywt_path)
+    for f in pywt_files:
+        basename = os.path.basename(f)
+        src = os.path.join(pywt_path, f)
+        if basename == 'init.py':
+            shutil.copy2(src, os.path.join(stage_dir, '__init__.py'))
+        else:
+            shutil.copy2(src, os.path.join(stage_dir, basename))
     os.chdir(stage_dir)
     sys.argv.append('--dist-dir=' + os.path.join('..', 'dist'))
 else:
-    sources = [ os.path.join(python_rel_dir, 'wiredtiger_wrap.c') ]
+    sources = [ os.path.join(conf_make_dir, python_rel_dir, 'CMakeFiles', '__wiredtiger.dir', 'wiredtigerPYTHON_wrap.c') ]
 
 wt_ext = Extension('_wiredtiger',
     sources = sources,
     extra_compile_args = cflags + cppflags,
     extra_link_args = ldflags,
     libraries = builtin_libraries,
-    extra_objects = [ os.path.join(build_dir, '.libs', 'libwiredtiger.a') ],
+    # As a temporary solution, we need to manually link the ext/compressor libraries. Unfortunately CMake's use of the builtin open
```

Comment:
"As a temporary solution" -- I'd remove this phrase.  Since the fix for this is in CMake, we don't know if it's temporary or if it will ever be done.  You could say "As a workaround", but then I would put the second sentence as the first sentence.

Reply:
Good point. I reworded the comment per your suggestion. I also added the ticket that will fix the CMake builtin support (tracked as a FIXME)